docs/inline.md

```diff
@@ -5,7 +5,7 @@ Scala 3 makes several improvements related to inlining:
 
 1. It introduces `inline` as a [soft keyword][soft-modifier].
 2. It introduces _inline expressions_.
-3. It guarantees that an _inline definition_ is inlined at the point of usage and an _inline expression_ is always reduced.
+3. It guarantees that an inlining actually happens instead of being best-effort.
```

Comment:
Much better.

Reply:
What about:

1. It introduces `inline` as a [soft keyword][soft-modifier].
2. It guarantees that inlining actually happens instead of being *best-effort*.
3. It introduces operations that are guaranteed to evaluate at compiletime.

Reply:
The last one is not specific. What about

> It introduces inline if-expressions and inline match-expressions.

Reply:
The point is not to be specific because the list is extremely large. It is not only the `inline-if` and `inline-match`, it also includes all operations in the `compiletime` package.

Reply:
It's an overview, I think it suffices to talk about the important items. `compiletime` package is indirectly related to inlining, if I were right.

Reply:
Some of the operations are directly related to inlining such as `summonInline`. The operations `inline if` and `summonInline` are both directly related to inlining, the difference is just that one needed extra syntax. Though now that I think about it, it looks like we could have all this logic in compile-time.